android/envoy/src/main/kotlin/org/greatfire/envoy/EnvoyState.kt

```diff
@@ -60,6 +60,8 @@ class EnvoyState private constructor() {
     // for debugging, simulate a connection timeout
     var debugTimeoutDriect = false
 
+    var debugMode = false
```

Comment:
can you comment what this means? I would not guess it means "only use one thread and don't shuffle the list of URLs", and I don't know what else it might do